roles/kubernetes-apps/helm/tasks/install_host.yml

```diff
@@ -34,9 +34,3 @@
   until: helm_task_result.rc == 0
   retries: 4
   delay: "{{ retry_stagger | random + 3 }}"
-
-- name: Helm | Copy socat wrapper for Container Linux
```

Comment:
Why are we removing socat?

Reply:
As written. Socat is part of Container Linux already. So no need

Reply:
Oh. I googled a bit and couldn't find that. Thanks for clarifying

Reply:
It was added in 1576.1.0 according to @MarkusTeufelberger :)